.github/actions/patch-nightly-version/patch-version.go

```diff
@@ -0,0 +1,42 @@
+package main
```

Comment:
Is there a reason we can't use the original version patching code? It feels weird to be injecting Golang code into a mostly JS/TS based project.

Reply:
Is there a specific command/script you are thinking of ? I grabbed this logic from the trigger nightly built from Gitlab repo